.github/workflows/gempush.yml

```diff
@@ -0,0 +1,28 @@
+name: Ruby Gem
+
+on:
+  push:
```

Comment:
maybe on new release?

Reply:
I think release is a subset of tag flow.